### PR TITLE
Simdna

### PR DESCRIPTION
**Pull request by lilleswing**
Simulated datasets of DNA

### CODE REVIEW

File: recipes/simdna/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/simdna/meta.yaml]: Please delete all comments originating from the example recipe.

File: recipes/simdna/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/deepchem/simdna.git
```
Comment (synapticarbors) [inline recipes/simdna/meta.yaml]: I'm wary of pulling from a different repo than what appears to be the official repo (https://github.com/kundajelab/simdna), although I'm not sure what the relationship is between the two. We don't want to cause confusion for users looking of something and getting another. 

Also, please see:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

File: recipes/simdna/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/simdna/meta.yaml]: `python` and `setuptools` appear to be the only `build` requirements. `numpy`, `scipy` and `matplotlib` are only `run` requirements.

**Comment by lilleswing** [comment]
all of these files besides recipes/simdna/meta.yml have been changed via merging upstream.  I can create a clean PR if the maintainers wish.

File: recipes/simdna/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    - scipy
+    - numpy >=1.9
+
+test:
+  imports:
+    - simdna
+
+about:
+  home: https://github.com/kundajelab/simdna
+  license: MIT
```
Comment (dougalsutherland) [inline recipes/simdna/meta.yaml]: MIT license requires a `license_file` too. You should file an issue about it not being in the distribution, or just send a PR to fix it (it's easy; [example](https://github.com/Ceasar/easywatch/pull/10)), add a comment here pointing to that, and in the meantime [package the license manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).

File: recipes/simdna/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  license_family: MIT
+  summary: 'Simulated datasets of DNA'
+
```
Comment (dougalsutherland) [inline recipes/simdna/meta.yaml]: Nit: you could delete this blank line.

**Comment by dougalsutherland** [comment]
Thanks @lilleswing!

Rather than waiting for Travis, since this is `noarch: python`, I just checked that it built okay on my Mac (it did), and merged after rebasing (for cleanliness).

A feedstock for this recipe should be generated soon, at https://github.com/conda-forge/simdna-feedstock/, and you'll get several automated messages about it. When the feedstock has been rendered, please take a look at the `README` in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Comment by dougalsutherland** [comment]
Guess github isn't smart enough to recognize that I merged the PR this way, so I'm going to hit the "close" button. The icon will show up as red, even though actually it should be green. :)

**Pull request closed without merging**